front/src/modules/people/utils/fieldsForPerson.tsx

```diff
@@ -1,5 +1,6 @@
 import { isValidPhoneNumber } from 'libphonenumber-js';
 
+import { Fields } from '@/spreadsheet-import/types';
```

Comment:
is it the wright import? I'm surprised that fieldsForPerson would depend on spreadheet types

Reply:
Yes it is used in `useSpreadsheetPersonImport`